README.md

```diff
@@ -54,21 +54,22 @@ sudo chown -R lorisadmin:lorisadmin /opt/$projectname
 cd /opt/$projectname/bin
 ```
 
-Get the code: Download the latest release from the 
-[releases page](https://github.com/aces/Loris-MRI/releases) 
+Get the code: Download the latest release from the
+[releases page](https://github.com/aces/Loris-MRI/releases)
 and extract it to `/opt/$projectname/bin/mri`
 
 #### 2. Install Python 3 with `pip` and `virtualenv`
 
 ```bash
-sudo apt install python3 
-sudo apt install python3-dev
+sudo apt install software-properties-common
+sudo add-apt-repository ppa:deadsnakes/ppa
```

Comment:
This repository allows us to not rely on the default distro repositories, which may be missing some packages we need (on Ubuntu 22 I only have 3.10 and 3.11 for instance).

This repository seems to be the standard for Python setups, example: 
https://askubuntu.com/questions/1398568/installing-python-who-is-deadsnakes-and-why-should-i-trust-them